src/browser/modules/EditorInfo/EditNodes.jsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * This model depicts the behaviour of the edit drawer that displays the node relationship
+ properties
+ */
+
+import React, { Component } from 'react'
+import {
+  Drawer,
+  DrawerBody,
+  DrawerHeader,
+  DrawerSection,
+  DrawerSectionBody,
+  DrawerSubHeader
+} from 'browser-components/drawer'
+
+import * as _ from 'lodash'
+import { getStringValue } from './utils'
+
+export class EditNodes extends Component {
+  render () {
+    let content = null
+    content = _.map(this.props.nodeProperties, (value, key) => {
+      return (
+        <div>
```

Comment:
add key attribute to this div 
key={key}

refer
https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Added key attribute